### ISSUE TITLE
Primal Trident crashed the server

### ISSUE THREAD

**Issue by Gabrielsyl20** [daedalus4096/PrimalMagick#285]
Hello. First of all, wow, thanks, this mod is the most amazing I ever played. I enjoy Minecraft again because of you.
I'm definetely joining the discord server.

NOW, for the problem : I right clicked on my trident to trow it at a Zombi coming my way, the game instantly crashed and the server also, I had to restart it all. When I came back, the trident was gone and I was still fighting the zombi.

**Comment by daedalus4096** [comment]
Hello! Thank you for the bug report, and I'm very happy to hear that you're enjoying the mod. As I haven't seen this bug myself, I need some more information from you before I can really investigate your crash. The most helpful thing would be the crash report from the Minecraft server that you were connected to; your server administrator should have access to this. If you can take that file, upload it to a site like [Pastebin](https://pastebin.com/), and then provide the resulting link here, that should give me the information necessary to start tracking down the source of this crash. The latest.log file from both the server and your client from the time of the crash would also be helpful, provided in the same way if possible.

**Comment by Zilgyd** [comment]
Hi, I have de same problem with the tridents, here the crash log report, is a forge server, I hope this log be useful: 
Note: I have others mod installed
Greetings!

---- Minecraft Crash Report ----
// Hey, that tickles! Hehehe!

Time: 2024-08-31 21:27:21
Description: Watching Server

java.lang.Error: ServerHangWatchdog detected that a single server tick took 60000004.00 seconds (should be max 0.05)
	at TRANSFORMER/minecraft@1.21/net.minecraft.server.dedicated.ServerWatchdog.run(ServerWatchdog.java:52) ~[forge-1.21-51.0.33-server.jar!/:?] {re:classloading}
	at java.base/java.lang.Thread.run(Unknown Source) [?:?] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server Watchdog
Suspected Mods: NONE
Stacktrace:
	at TRANSFORMER/minecraft@1.21/net.minecraft.server.dedicated.ServerWatchdog.run(ServerWatchdog.java:52) ~[forge-1.21-51.0.33-server.jar!/:?] {re:classloading}
-- Thread Dump --
Details:
	Threads: "Reference Handler" daemon prio=10 Id=9 RUNNABLE
	at java.base@21.0.4/java.lang.ref.Reference.waitForReferencePendingList(Native Method)
	at java.base@21.0.4/java.lang.ref.Reference.processPendingReferences(Unknown Source)
	at java.base@21.0.4/java.lang.ref.Reference$ReferenceHandler.run(Unknown Source)


"Finalizer" daemon prio=8 Id=10 WAITING on java.lang.ref.NativeReferenceQueue$Lock@171172dd
	at java.base@21.0.4/java.lang.Object.wait0(Native Method)
	-  waiting on java.lang.ref.NativeReferenceQueue$Lock@171172dd
	at java.base@21.0.4/java.lang.Object.wait(Unknown Source)
	at java.base@21.0.4/java.lang.Object.wait(Unknown Source)
	at java.base@21.0.4/java.lang.ref.NativeReferenceQueue.await(Unknown Source)
	at java.base@21.0.4/java.lang.ref.ReferenceQueue.remove0(Unknown Source)
	at java.base@21.0.4/java.lang.ref.NativeReferenceQueue.remove(Unknown Source)
	at java.base@21.0.4/java.lang.ref.Finalizer$FinalizerThread.run(Unknown Source)


"Signal Dispatcher" daemon prio=9 Id=11 RUNNABLE


"Notification Thread" daemon prio=9 Id=16 RUNNABLE


"Common-Cleaner" daemon prio=8 Id=17 TIMED_WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@19d69e4f
	at java.base@21.0.4/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@19d69e4f
	at java.base@21.0.4/java.util.concurrent.locks.LockSupport.parkNanos(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
	at java.base@21.0.4/java.lang.ref.ReferenceQueue.await(Unknown Source)
	at java.base@21.0.4/java.lang.ref.ReferenceQueue.remove0(Unknown Source)
	at java.base@21.0.4/java.lang.ref.ReferenceQueue.remove(Unknown Source)
	at java.base@21.0.4/jdk.internal.ref.CleanerImpl.run(Unknown Source)
	at java.base@21.0.4/java.lang.Thread.runWith(Unknown Source)
	...


"JNA Cleaner" daemon prio=5 Id=23 TIMED_WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@1de919ad
	at java.base@21.0.4/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@1de919ad
	at java.base@21.0.4/java.util.concurrent.locks.LockSupport.parkNanos(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
	at java.base@21.0.4/java.lang.ref.ReferenceQueue.await(Unknown Source)
	at java.base@21.0.4/java.lang.ref.ReferenceQueue.remove0(Unknown Source)
	at java.base@21.0.4/java.lang.ref.ReferenceQueue.remove(Unknown Source)
	at SECURE-BOOTSTRAP/com.sun.jna@5.14.0/com.sun.jna.internal.Cleaner$CleanerThread.run(Cleaner.java:154)


"Timer hack thread" daemon prio=5 Id=27 TIMED_WAITING
	at java.base@21.0.4/java.lang.Thread.sleep0(Native Method)
	at java.base@21.0.4/java.lang.Thread.sleep(Unknown Source)
	at TRANSFORMER/minecraft@1.21/net.minecraft.Util$7.run(Util.java:756)


"FileSystemWatchService" daemon prio=5 Id=31 RUNNABLE (in native)
	at java.base@21.0.4/sun.nio.fs.LinuxWatchService.poll(Native Method)
	at java.base@21.0.4/sun.nio.fs.LinuxWatchService$Poller.run(Unknown Source)
	at java.base@21.0.4/java.lang.Thread.runWith(Unknown Source)
	at java.base@21.0.4/java.lang.Thread.run(Unknown Source)


"Config Lib config file watcher" daemon prio=5 Id=32 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@58eb9474
	at java.base@21.0.4/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@58eb9474
	at java.base@21.0.4/java.util.concurrent.locks.LockSupport.park(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ForkJoinPool.unmanagedBlock(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ForkJoinPool.managedBlock(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.LinkedBlockingDeque.takeFirst(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.LinkedBlockingDeque.take(Unknown Source)
	...


"FileSystemWatchService" daemon prio=5 Id=33 RUNNABLE (in native)
	at java.base@21.0.4/sun.nio.fs.LinuxWatchService.poll(Native Method)
	at java.base@21.0.4/sun.nio.fs.LinuxWatchService$Poller.run(Unknown Source)
	at java.base@21.0.4/java.lang.Thread.runWith(Unknown Source)
	at java.base@21.0.4/java.lang.Thread.run(Unknown Source)


"config-file-watcher-0-0" daemon prio=5 Id=34 TIMED_WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@4470822e
	at java.base@21.0.4/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@4470822e
	at java.base@21.0.4/java.util.concurrent.locks.LockSupport.parkNanos(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.LinkedBlockingDeque.pollFirst(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.LinkedBlockingDeque.poll(Unknown Source)
	at java.base@21.0.4/sun.nio.fs.AbstractWatchService.poll(Unknown Source)
	at SECURE-BOOTSTRAP/com.electronwill.nightconfig.core@3.7.3/com.electronwill.nightconfig.core.file.FileWatcher$FsWatcher.run(FileWatcher.java:376)
	at java.base@21.0.4/java.lang.Thread.runWith(Unknown Source)
	...


"pool-3-thread-1" prio=5 Id=35 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@69f94809
	at java.base@21.0.4/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@69f94809
	at java.base@21.0.4/java.util.concurrent.locks.LockSupport.park(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ForkJoinPool.unmanagedBlock(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ForkJoinPool.managedBlock(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	...


"ConfigSaver" daemon prio=5 Id=36 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3c8800ea
	at java.base@21.0.4/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@3c8800ea
	at java.base@21.0.4/java.util.concurrent.locks.LockSupport.park(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ForkJoinPool.unmanagedBlock(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ForkJoinPool.managedBlock(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.LinkedBlockingQueue.take(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
	...


"ConfigSaver" daemon prio=5 Id=38 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@38c46b3b
	at java.base@21.0.4/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@38c46b3b
	at java.base@21.0.4/java.util.concurrent.locks.LockSupport.park(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ForkJoinPool.unmanagedBlock(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ForkJoinPool.managedBlock(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.LinkedBlockingQueue.take(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
	...


"Yggdrasil Key Fetcher" daemon prio=5 Id=42 TIMED_WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7865efff
	at java.base@21.0.4/jdk.internal.misc.Unsafe.park(Native Method)
	-  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@7865efff
	at java.base@21.0.4/java.util.concurrent.locks.LockSupport.parkNanos(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base@21.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	...


"DestroyJavaVM" prio=5 Id=50 RUNNABLE


"Netty Epoll Server IO #0" daemon prio=5 Id=52 RUNNABLE (in native)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.Native.epollWait(Native Method)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.Native.epollWait(Native.java:209)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.Native.epollWait(Native.java:202)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.EpollEventLoop.epollWaitNoTimerChange(EpollEventLoop.java:316)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:373)
	at SECURE-BOOTSTRAP/io.netty.common@4.1.97.Final/io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at SECURE-BOOTSTRAP/io.netty.common@4.1.97.Final/io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at java.base@21.0.4/java.lang.Thread.runWith(Unknown Source)
	...


"Server Watchdog" daemon prio=5 Id=59 RUNNABLE
	at java.management@21.0.4/sun.management.ThreadImpl.dumpThreads0(Native Method)
	at java.management@21.0.4/sun.management.ThreadImpl.dumpAllThreads(Unknown Source)
	at java.management@21.0.4/sun.management.ThreadImpl.dumpAllThreads(Unknown Source)
	at TRANSFORMER/minecraft@1.21/net.minecraft.server.dedicated.ServerWatchdog.run(ServerWatchdog.java:50)
	at java.base@21.0.4/java.lang.Thread.runWith(Unknown Source)
	at java.base@21.0.4/java.lang.Thread.run(Unknown Source)


"Netty Epoll Server IO #1" daemon prio=5 Id=61 RUNNABLE (in native)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.Native.epollWait(Native Method)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.Native.epollWait(Native.java:209)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.Native.epollWait(Native.java:202)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.EpollEventLoop.epollWaitNoTimerChange(EpollEventLoop.java:316)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:373)
	at SECURE-BOOTSTRAP/io.netty.common@4.1.97.Final/io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at SECURE-BOOTSTRAP/io.netty.common@4.1.97.Final/io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at java.base@21.0.4/java.lang.Thread.runWith(Unknown Source)
	...


"Netty Epoll Server IO #2" daemon prio=5 Id=62 RUNNABLE (in native)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.Native.epollWait(Native Method)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.Native.epollWait(Native.java:209)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.Native.epollWait(Native.java:202)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.EpollEventLoop.epollWaitNoTimerChange(EpollEventLoop.java:316)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:373)
	at SECURE-BOOTSTRAP/io.netty.common@4.1.97.Final/io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at SECURE-BOOTSTRAP/io.netty.common@4.1.97.Final/io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at java.base@21.0.4/java.lang.Thread.runWith(Unknown Source)
	...


"Netty Epoll Server IO #3" daemon prio=5 Id=63 RUNNABLE (in native)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.Native.epollWait(Native Method)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.Native.epollWait(Native.java:209)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.Native.epollWait(Native.java:202)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.EpollEventLoop.epollWaitNoTimerChange(EpollEventLoop.java:316)
	at SECURE-BOOTSTRAP/io.netty.transport.classes.epoll@4.1.97.Final/io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:373)
	at SECURE-BOOTSTRAP/io.netty.common@4.1.97.Final/io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at SECURE-BOOTSTRAP/io.netty.common@4.1.97.Final/io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at java.base@21.0.4/java.lang.Thread.runWith(Unknown Source)
	...



Stacktrace:
	at TRANSFORMER/minecraft@1.21/net.minecraft.server.dedicated.ServerWatchdog.run(ServerWatchdog.java:52) ~[forge-1.21-51.0.33-server.jar!/:?] {re:classloading}
	at java.base/java.lang.Thread.run(Unknown Source) [?:?] {}


-- Performance stats --
Details:
	Random tick rate: 3
	Level stats: ResourceKey[minecraft:dimension / minecraft:overworld]: players: 1, entities: 1,1,1,0,0,0,674 [minecraft:player:1], block_entities: 629 [<null>:629], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 1,1,1,0,0,0,674,
ResourceKey[minecraft:dimension / minecraft:the_nether]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / minecraft:the_end]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0,
ResourceKey[minecraft:dimension / cavernous:the_myst]: players: 0, entities: 0,0,0,0,0,0,0 [], block_entities: 0 [], block_ticks: 0, fluid_ticks: 0, chunk_source: Chunks[S] W: 0 E: 0,0,0,0,0,0,0
Stacktrace:
	at TRANSFORMER/minecraft@1.21/net.minecraft.server.dedicated.ServerWatchdog.run(ServerWatchdog.java:52) ~[forge-1.21-51.0.33-server.jar!/:?] {re:classloading}
	at java.base/java.lang.Thread.run(Unknown Source) [?:?] {}


-- System Details --
Details:
	Minecraft Version: 1.21
	Minecraft Version ID: 1.21
	Operating System: Linux (amd64) version 6.8.0-40-generic
	Java Version: 21.0.4, Eclipse Adoptium
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Eclipse Adoptium
	Memory: 308097312 bytes (293 MiB) / 1644167168 bytes (1568 MiB) up to 5242880000 bytes (5000 MiB)
	CPUs: 3
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 7 5800X 8-Core Processor
	Identifier: AuthenticAMD Family 25 Model 33 Stepping 0
	Microarchitecture: Zen 3
	Frequency (GHz): -0.00
	Number of physical packages: 3
	Number of physical CPUs: 3
	Number of logical CPUs: 3
	Graphics card #0 name: unknown
	Graphics card #0 vendor: unknown
	Graphics card #0 VRAM (MiB): 0.00
	Graphics card #0 deviceId: unknown
	Graphics card #0 versionInfo: unknown
	Virtual memory max (MiB): 3277.85
	Virtual memory used (MiB): 2688.10
	Swap memory total (MiB): 0.00
	Swap memory used (MiB): 0.00
	Space in storage for jna.tmpdir (MiB): <path not set>
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): <path not set>
	Space in storage for io.netty.native.workdir (MiB): <path not set>
	Space in storage for java.io.tmpdir (MiB): available: 54804.10, total: 58446.24
	Space in storage for workdir (MiB): available: 54804.10, total: 58446.24
	JVM Flags: 2 total; -Xmx5000M -Xms1024M
	Server Running: true
	Player Count: 1 / 7; [ServerPlayer['Zilgyd'/261, l='ServerLevel[world]', x=-506.23, y=-21.00, z=-59.94]]
	Active Data Packs: vanilla, mod:mr_dungeons_andtavernsnetherfortressoverhaul, mod:fullbrightnesstoggle, mod:kuma_api (incompatible), mod:overhauledstructures (incompatible), mod:supermartijn642configlib (incompatible), mod:mr_dungeons_andtavernswoodlandmansionoverhaul, mod:treeharvester, mod:beo, mod:jei (incompatible), mod:breadvariants (incompatible), mod:minersdream (incompatible), mod:sound_physics_remastered (incompatible), mod:monsterplus (incompatible), mod:waystones (incompatible), mod:mr_dungeons_andtavernsancientcityoverhaul, mod:structory, mod:moreconcrete (incompatible), mod:cookingforblockheads (incompatible), mod:xaeroworldmap (incompatible), mod:comforts (incompatible), mod:travelersbackpack (incompatible), mod:badpackets (incompatible), mod:mr_kawamoods_waystonesinshrinesofdungeonsandtaverns, mod:primalmagick (incompatible), mod:mixinextras (incompatible), mod:glitchcore (incompatible), mod:elytraaccessory (incompatible), mod:medievalend (incompatible), mod:hopour, mod:mr_dungeons_andtavernsjungletempleoverhaul, mod:wider_ender_chests (incompatible), mod:corail_spawners (incompatible), mod:balm (incompatible), mod:shulkerbox (incompatible), mod:morevanillashields (incompatible), mod:macevariants (incompatible), mod:reinforcedobsidian (incompatible), mod:forge (incompatible), mod:uniquemagic (incompatible), mod:mr_dungeons_andtavernsswamphutoverhaul, mod:durabilitytooltip (incompatible), mod:replantingcrops, mod:emerald (incompatible), mod:trade_cycling (incompatible), mod:terrablender (incompatible), mod:biomesoplenty (incompatible), mod:mr_dungeons_andtavernsstrongholdrework, mod:unwrecked_ships, mod:hoporp, mod:totemaccessory (incompatible), mod:ohmega (incompatible), mod:mr_dungeons_andtavernsoceanmonumentoverhaul, mod:sunsembrace (incompatible), mod:spectrelib (incompatible), mod:enchantmenttransfer (incompatible), mod:wthit (incompatible), mod:corail_woodcutter (incompatible), mod:advancedcombat (incompatible), mod:xaerominimap (incompatible), mod:shulkerdropstwo, mod:craftableenchantedgoldenapple (incompatible), mod:collective, mod:advancednetherite (incompatible), mod:storagedrawers (incompatible), mod:nullscape, mod:mr_dungeons_andtaverns, mod:endreborn (incompatible), mod:serversleep_mr (incompatible), mod:bedrockplus (incompatible), mod:resource_nether_ores (incompatible), mod:tombstone (incompatible), mod:mr_dungeons_andtavernspillageroutpostoverhaul, mod:resourcelibrary (incompatible), mod:mr_dungeons_andtavernsdeserttempleoverhaul, mod:passiveshield, mod:lios_overhauled_villages (incompatible), mod:mr_dungeons_andtavernsendcastlestandalone, mod:refurbished_furniture (incompatible), mod:monolib (incompatible), mod:golden_foods (incompatible), mod:more_bows_and_arrows (incompatible), mod:framework (incompatible), mod:allarrowsinfinityfix (incompatible), mod:hopo, mod:smallernetherportals, mod:anzuslavawalker (incompatible), mod:corail_pillar (incompatible), mod:responsiveshields (incompatible), mod:gd_betterloot, file/cavernous-r-v2-2-0.zip, mod:vtaw_mw (incompatible)
	Available Data Packs: bundle, file/cavernous-r-v2-2-0.zip, mod:advancedcombat (incompatible), mod:advancednetherite (incompatible), mod:allarrowsinfinityfix (incompatible), mod:anzuslavawalker (incompatible), mod:badpackets (incompatible), mod:balm (incompatible), mod:bedrockplus (incompatible), mod:beo, mod:biomesoplenty (incompatible), mod:breadvariants (incompatible), mod:collective, mod:comforts (incompatible), mod:cookingforblockheads (incompatible), mod:corail_pillar (incompatible), mod:corail_spawners (incompatible), mod:corail_woodcutter (incompatible), mod:craftableenchantedgoldenapple (incompatible), mod:durabilitytooltip (incompatible), mod:elytraaccessory (incompatible), mod:emerald (incompatible), mod:enchantmenttransfer (incompatible), mod:endreborn (incompatible), mod:forge (incompatible), mod:framework (incompatible), mod:fullbrightnesstoggle, mod:gd_betterloot, mod:glitchcore (incompatible), mod:golden_foods (incompatible), mod:hopo, mod:hoporp, mod:hopour, mod:jei (incompatible), mod:kuma_api (incompatible), mod:lios_overhauled_villages (incompatible), mod:macevariants (incompatible), mod:medievalend (incompatible), mod:minersdream (incompatible), mod:mixinextras (incompatible), mod:monolib (incompatible), mod:monsterplus (incompatible), mod:more_bows_and_arrows (incompatible), mod:moreconcrete (incompatible), mod:morevanillashields (incompatible), mod:mr_dungeons_andtaverns, mod:mr_dungeons_andtavernsancientcityoverhaul, mod:mr_dungeons_andtavernsdeserttempleoverhaul, mod:mr_dungeons_andtavernsendcastlestandalone, mod:mr_dungeons_andtavernsjungletempleoverhaul, mod:mr_dungeons_andtavernsnetherfortressoverhaul, mod:mr_dungeons_andtavernsoceanmonumentoverhaul, mod:mr_dungeons_andtavernspillageroutpostoverhaul, mod:mr_dungeons_andtavernsstrongholdrework, mod:mr_dungeons_andtavernsswamphutoverhaul, mod:mr_dungeons_andtavernswoodlandmansionoverhaul, mod:mr_kawamoods_waystonesinshrinesofdungeonsandtaverns, mod:nullscape, mod:ohmega (incompatible), mod:overhauledstructures (incompatible), mod:passiveshield, mod:primalmagick (incompatible), mod:refurbished_furniture (incompatible), mod:reinforcedobsidian (incompatible), mod:replantingcrops, mod:resource_nether_ores (incompatible), mod:resourcelibrary (incompatible), mod:responsiveshields (incompatible), mod:serversleep_mr (incompatible), mod:shulkerbox (incompatible), mod:shulkerdropstwo, mod:smallernetherportals, mod:sound_physics_remastered (incompatible), mod:spectrelib (incompatible), mod:storagedrawers (incompatible), mod:structory, mod:sunsembrace (incompatible), mod:supermartijn642configlib (incompatible), mod:terrablender (incompatible), mod:tombstone (incompatible), mod:totemaccessory (incompatible), mod:trade_cycling (incompatible), mod:travelersbackpack (incompatible), mod:treeharvester, mod:uniquemagic (incompatible), mod:unwrecked_ships, mod:vtaw_mw (incompatible), mod:waystones (incompatible), mod:wider_ender_chests (incompatible), mod:wthit (incompatible), mod:xaerominimap (incompatible), mod:xaeroworldmap (incompatible), trade_rebalance, vanilla
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Stable
	World Seed: 1073756039506960466
	Is Modded: Definitely; Server brand changed to 'forge'
	Type: Dedicated Server (map_server.txt)
	ModLauncher: 10.2.1
	ModLauncher launch target: forge_server
	ModLauncher naming: mcp
	ModLauncher services: 
		/ slf4jfixer PLUGINSERVICE 
		/ runtimedistcleaner PLUGINSERVICE 
		/ runtime_enum_extender PLUGINSERVICE 
		/ object_holder_definalize PLUGINSERVICE 
		/ capability_token_subclass PLUGINSERVICE 
		/ accesstransformer PLUGINSERVICE 
		/ eventbus PLUGINSERVICE 
		/ mixin PLUGINSERVICE 
		/ fml TRANSFORMATIONSERVICE 
		/ mixin TRANSFORMATIONSERVICE 
	FML Language Providers: 
		minecraft@1.0
		lowcodefml@51
		javafml@51.0.33
	Mod List: 
		DnT-nether-fortress-overhaul-v2.1 [Forge].jar     |Dungeons and Taverns Nether Fo|mr_dungeons_andtavernsnetherfo|1-v2.1              |DONE      |Manifest: NOSIGNATURE
		fullbrightnesstoggle-1.21.1-4.2.jar               |Full Brightness Toggle        |fullbrightnesstoggle          |4.2                 |DONE      |Manifest: NOSIGNATURE
		kuma-api-forge-21.0.4+1.21.jar                    |KumaAPI                       |kuma_api                      |21.0.4              |DONE      |Manifest: NOSIGNATURE
		Epic Dungeons-0.1.04-Forge-1.21.jar               |Epic Dungeons                 |overhauledstructures          |0.1.04-Forge-1.21   |DONE      |Manifest: NOSIGNATURE
		supermartijn642configlib-1.1.8-forge-mc1.21.jar   |SuperMartijn642's Config Libra|supermartijn642configlib      |1.1.8               |DONE      |Manifest: NOSIGNATURE
		DnT-woodland-mansion-overhaul-v1.1 [Forge].jar    |Dungeons and Taverns Woodland |mr_dungeons_andtavernswoodland|1-v1.1              |DONE      |Manifest: NOSIGNATURE
		treeharvester-1.21.1-9.1.jar                      |Tree Harvester                |treeharvester                 |9.1                 |DONE      |Manifest: NOSIGNATURE
		BasicEndOres-forge-1.21-5.0.2.jar                 |BasicEndOres                  |beo                           |5.0.2               |DONE      |Manifest: NOSIGNATURE
		jei-1.21-forge-19.8.2.99.jar                      |Just Enough Items             |jei                           |19.8.2.99           |DONE      |Manifest: NOSIGNATURE
		breadvariants-forge-1.21-1.0.2.jar                |Bread Variants                |breadvariants                 |1.0.2               |DONE      |Manifest: NOSIGNATURE
		minersdream-1.21-0.1.jar                          |Miners Dream                  |minersdream                   |0.1                 |DONE      |Manifest: NOSIGNATURE
		sound-physics-remastered-forge-1.21.1-1.4.6.jar   |Sound Physics Remastered      |sound_physics_remastered      |1.21.1-1.4.6        |DONE      |Manifest: NOSIGNATURE
		MonsterPlus-Forge1.21.1-v1.1.8.2b.jar             |Monster Plus                  |monsterplus                   |1.1.8.2             |DONE      |Manifest: NOSIGNATURE
		waystones-forge-1.21.1-21.1.1.jar                 |Waystones                     |waystones                     |21.1.1              |DONE      |Manifest: NOSIGNATURE
		DnT-ancient-city-overhaul-v2 [Forge].jar          |Dungeons and Taverns Ancient C|mr_dungeons_andtavernsancientc|1-v2                |DONE      |Manifest: NOSIGNATURE
		Structory_1.21_v1.3.5.jar                         |Structory                     |structory                     |1.3.5               |DONE      |Manifest: NOSIGNATURE
		moreconcrete-forge-1.5.2-1.21.jar                 |More Concrete                 |moreconcrete                  |1.5.2               |DONE      |Manifest: NOSIGNATURE
		cookingforblockheads-forge-1.21-21.0.4.jar        |Cooking for Blockheads        |cookingforblockheads          |21.0.4              |DONE      |Manifest: NOSIGNATURE
		XaerosWorldMap_1.39.0_Forge_1.21.jar              |Xaero's World Map             |xaeroworldmap                 |1.39.0              |DONE      |Manifest: NOSIGNATURE
		comforts-forge-9.0.0+1.21.jar                     |Comforts                      |comforts                      |9.0.0+1.21          |DONE      |Manifest: NOSIGNATURE
		travelersbackpack-forge-1.21-10.0.1.jar           |Traveler's Backpack           |travelersbackpack             |10.0.1              |DONE      |Manifest: NOSIGNATURE
		badpackets-forge-0.8.1.jar                        |Bad Packets                   |badpackets                    |0.8.1               |DONE      |Manifest: NOSIGNATURE
		kawamoods-waystones-in-shrines-of-dungeons-and-tav|KawaMood's Waystones in Shrine|mr_kawamoods_waystonesinshrine|1.0.3               |DONE      |Manifest: NOSIGNATURE
		primalmagick-5.0.0-beta.jar                       |Primal Magick                 |primalmagick                  |5.0.0-beta          |DONE      |Manifest: NOSIGNATURE
		mixinextras-forge-0.3.5.jar                       |MixinExtras                   |mixinextras                   |0.3.5               |DONE      |Manifest: NOSIGNATURE
		GlitchCore-forge-1.21-2.0.0.2.jar                 |GlitchCore                    |glitchcore                    |2.0.0.2             |DONE      |Manifest: NOSIGNATURE
		elytraaccessory-1.0.0+forge-1.21-1.21.1.jar       |Elytra Accessory              |elytraaccessory               |1.0.0               |DONE      |Manifest: NOSIGNATURE
		forge_medieval_buildings_end_edition-1.0.2-1.21.ja|Medieval Buildings [The End Ed|medievalend                   |1.0.2               |DONE      |Manifest: NOSIGNATURE
		HopoBetterUnderwaterRuins-[1.21]-1.2.0b.jar       |HopoBetterUnderwaterRuins     |hopour                        |1.2.0               |DONE      |Manifest: NOSIGNATURE
		DnT-jungle-temple-overhaul-v1.1 [Forge].jar       |Dungeons and Taverns Jungle Te|mr_dungeons_andtavernsjunglete|1-v1.1              |DONE      |Manifest: NOSIGNATURE
		forge_wider_ender_chests-1.0.2-1.21.jar           |Wider Ender Chests            |wider_ender_chests            |1.0.2               |DONE      |Manifest: NOSIGNATURE
		Corail-Spawners-1.21-040.jar                      |Corail Spawners               |corail_spawners               |1.21-040            |DONE      |Manifest: NOSIGNATURE
		balm-forge-1.21-21.0.14-all.jar                   |Balm                          |balm                          |21.0.14             |DONE      |Manifest: NOSIGNATURE
		shulkerbox-forge-1.21.1-2.0.5.jar                 |Advanced Shulkerboxes         |shulkerbox                    |1.21.1-2.0.5        |DONE      |Manifest: NOSIGNATURE
		morevanillashields-1.0.1-1.21.jar                 |More Vanilla Shields          |morevanillashields            |1.0.1-1.21          |DONE      |Manifest: NOSIGNATURE
		macevariants-forge-1.21-1.0.6.jar                 |MaceVariants                  |macevariants                  |1.0.6               |DONE      |Manifest: NOSIGNATURE
		reinforcedobsidian-1.21-0.1.jar                   |ReinforcedObsidian            |reinforcedobsidian            |1.21-0.1            |DONE      |Manifest: NOSIGNATURE
		forge-1.21-51.0.33-universal.jar                  |Forge                         |forge                         |51.0.33             |DONE      |Manifest: NOSIGNATURE
		uniquemagic-1.21-1.5.0.jar                        |Unique Magic                  |uniquemagic                   |1.21-1.5.0          |DONE      |Manifest: NOSIGNATURE
		DnT-swamp-hut-overhaul-v2 [Forge].jar             |Dungeons and Taverns Swamp Hut|mr_dungeons_andtavernsswamphut|1-v2                |DONE      |Manifest: NOSIGNATURE
		durabilitytooltip-1.1.5-forge-mc1.21.jar          |Durability Tooltip            |durabilitytooltip             |1.1.5               |DONE      |Manifest: NOSIGNATURE
		replantingcrops-1.21.0-5.4.jar                    |Replanting Crops              |replantingcrops               |5.4                 |DONE      |Manifest: NOSIGNATURE
		Easy Emerald-forge-1.21-3.4.5.jar                 |Easy Emerald                  |emerald                       |3.4.5               |DONE      |Manifest: NOSIGNATURE
		forge-1.21-51.0.33-server.jar                     |Minecraft                     |minecraft                     |1.21                |DONE      |Manifest: NOSIGNATURE
		trade-cycling-forge-1.21.1-1.0.14.jar             |Trade Cycling                 |trade_cycling                 |1.21.1-1.0.14       |DONE      |Manifest: NOSIGNATURE
		TerraBlender-forge-1.21-4.0.0.1.jar               |TerraBlender                  |terrablender                  |4.0.0.1             |DONE      |Manifest: NOSIGNATURE
		BiomesOPlenty-forge-1.21-21.0.0.15.jar            |Biomes O' Plenty              |biomesoplenty                 |21.0.0.15           |DONE      |Manifest: NOSIGNATURE
		vtaw_mw-forge-1.21-1.0.4.jar                      |Variant Tools and Weaponry - E|vtaw_mw                       |1.0.4               |DONE      |Manifest: NOSIGNATURE
		DnT-stronghold-rework-v2.1 [Forge].jar            |Dungeons and Taverns Stronghol|mr_dungeons_andtavernsstrongho|1-v2.1              |DONE      |Manifest: NOSIGNATURE
		unwrecked-ships-plus-2.0.0-forge.jar              |Unwrecked Ships               |unwrecked_ships               |2.0.0               |DONE      |Manifest: NOSIGNATURE
		HopoBetterRuinedPortals-[1.21]-1.4.3.jar          |HopoBetterRuinedPortals       |hoporp                        |1.4.3               |DONE      |Manifest: NOSIGNATURE
		totemaccessory-1.0.0+forge-1.21-1.21.1.jar        |Totem Accessory               |totemaccessory                |1.0.0               |DONE      |Manifest: NOSIGNATURE
		ohmega-1.2.2+forge-1.21-1.21.1.jar                |Ohmega                        |ohmega                        |1.2.2               |DONE      |Manifest: NOSIGNATURE
		DnT-ocean-monument-overhaul-v1.1 [Forge].jar      |Dungeons and Taverns Ocean Mon|mr_dungeons_andtavernsoceanmon|1-v1.1              |DONE      |Manifest: NOSIGNATURE
		sunsembrace-1.21.0-1.0.0.jar                      |Sun's Embrace                 |sunsembrace                   |1.21.0-1.0.0        |DONE      |Manifest: NOSIGNATURE
		spectrelib-forge-0.17.1+1.21.jar                  |SpectreLib                    |spectrelib                    |0.17.1+1.21         |DONE      |Manifest: NOSIGNATURE
		enchantmenttransfer-0.0.6-1.21.jar                |Enchantment Transfer          |enchantmenttransfer           |0.0.6               |DONE      |Manifest: NOSIGNATURE
		wthit-forge-12.3.0.jar                            |wthit                         |wthit                         |12.3.0              |DONE      |Manifest: NOSIGNATURE
		corail_woodcutter-1.21.3.4.1.jar                  |Corail Woodcutter             |corail_woodcutter             |3.4.1               |DONE      |Manifest: NOSIGNATURE
		AC-Revitalized-1.21-4.3FE.jar                     |Advanced Combat (Revitalized) |advancedcombat                |4.3                 |DONE      |Manifest: NOSIGNATURE
		Xaeros_Minimap_24.3.0_Forge_1.21.jar              |Xaero's Minimap               |xaerominimap                  |24.3.0              |DONE      |Manifest: NOSIGNATURE
		shulkerdropstwo-1.21.1-3.4.jar                    |Shulker Drops Two             |shulkerdropstwo               |3.4                 |DONE      |Manifest: NOSIGNATURE
		craftableenchantedgoldenapple-1.21-0.1.jar        |CraftableEnchantedGoldenApple |craftableenchantedgoldenapple |1.21-0.1            |DONE      |Manifest: NOSIGNATURE
		collective-1.21.1-7.84.jar                        |Collective                    |collective                    |7.84                |DONE      |Manifest: NOSIGNATURE
		advancednetherite-forge-2.1.6-1.21.jar            |Advanced Netherite            |advancednetherite             |2.1.6               |DONE      |Manifest: NOSIGNATURE
		StorageDrawers-1.21-Forge-13.2.0.jar              |Storage Drawers               |storagedrawers                |13.2.0              |DONE      |Manifest: NOSIGNATURE
		Nullscape_1.21_v1.2.7.jar                         |Nullscape                     |nullscape                     |1.2.7               |DONE      |Manifest: NOSIGNATURE
		dungeons-and-taverns-v4.3 [Forge].jar             |Dungeons and Taverns          |mr_dungeons_andtaverns        |1-v4.3              |DONE      |Manifest: NOSIGNATURE
		endorium-forge-1.1.5.jar                          |Endorium                      |endreborn                     |1.1.5               |DONE      |Manifest: NOSIGNATURE
		serversleep-datapack.jar                          |ServerSleep                   |serversleep_mr                |1-datapack          |DONE      |Manifest: NOSIGNATURE
		Bedrock Plus Mod-1.21-1.6.jar                     |Bedrock Plus Mod              |bedrockplus                   |1.6                 |DONE      |Manifest: NOSIGNATURE
		resource_nether_ores-forge-1.21-2.0.4.jar         |Resource Nether Ores          |resource_nether_ores          |2.0.4               |DONE      |Manifest: NOSIGNATURE
		tombstone-1.21.1-9.0.5.jar                        |Corail Tombstone              |tombstone                     |9.0.5               |DONE      |Manifest: NOSIGNATURE
		DnT-pillager-outpost-overhaul-v2.2 [Forge].jar    |Dungeons and Taverns Pillager |mr_dungeons_andtavernspillager|1-v2.2              |DONE      |Manifest: NOSIGNATURE
		resourcelibrary-forge-1.21-2.0.2.jar              |Resource Library              |resourcelibrary               |1.21-2.0.2          |DONE      |Manifest: NOSIGNATURE
		DnT-desert-temple-overhaul-v1.1 [Forge].jar       |Dungeons and Taverns Desert Te|mr_dungeons_andtavernsdesertte|1-v1.1              |DONE      |Manifest: NOSIGNATURE
		passiveshield-1.21.1-3.6.jar                      |Passive Shield                |passiveshield                 |3.6                 |DONE      |Manifest: NOSIGNATURE
		lios_overhauled_villages-1.18.2-1.21-v0.0.4.jar   |Lio's Overhauled Villages     |lios_overhauled_villages      |0.0.4-1.18.2-1.21-Un|DONE      |Manifest: NOSIGNATURE
		DnT-end-castle-standalone-v1 [Forge].jar          |Dungeons and Taverns End Castl|mr_dungeons_andtavernsendcastl|1-v1                |DONE      |Manifest: NOSIGNATURE
		refurbished_furniture-forge-1.21-1.0.6.jar        |MrCrayfish's Furniture Mod: Re|refurbished_furniture         |1.0.6               |DONE      |Manifest: 0D:78:5F:44:C0:47:0C:8C:E2:63:A3:04:43:D4:12:7D:B0:7C:35:37:DC:40:B1:C1:98:EC:51:EB:3B:3C:45:99
		monolib-forge-1.21-1.3.0.jar                      |MonoLib                       |monolib                       |1.3.0               |DONE      |Manifest: NOSIGNATURE
		golden_foods-forge-1.21-2.3.0.jar                 |Golden Foods                  |golden_foods                  |2.3.0               |DONE      |Manifest: NOSIGNATURE
		more_bows_and_arrows-forge-1.21-3.1.0.jar         |More Bows and Arrows          |more_bows_and_arrows          |3.1.0               |DONE      |Manifest: NOSIGNATURE
		framework-forge-1.21-0.9.0.jar                    |Framework                     |framework                     |0.9.0               |DONE      |Manifest: 0D:78:5F:44:C0:47:0C:8C:E2:63:A3:04:43:D4:12:7D:B0:7C:35:37:DC:40:B1:C1:98:EC:51:EB:3B:3C:45:99
		allarrowsinfinityfix-1.7.1-forge-mc1.21.jar       |All Arrows Infinity Fix       |allarrowsinfinityfix          |1.7.1               |DONE      |Manifest: NOSIGNATURE
		HopoBetterMineshaft-[1.21]-1.2.7.jar              |HopoBetterMineshaft           |hopo                          |1.2.7               |DONE      |Manifest: NOSIGNATURE
		smallernetherportals-1.21.1-3.8.jar               |Smaller Nether Portals        |smallernetherportals          |3.8                 |DONE      |Manifest: NOSIGNATURE
		anzuslavawalker-1.0.0-1.21.jar                    |Anzu's Lava Walker            |anzuslavawalker               |1.0.0-1.21          |DONE      |Manifest: NOSIGNATURE
		corail_pillar-6.4.0.jar                           |Corail Pillar                 |corail_pillar                 |6.4.0               |DONE      |Manifest: NOSIGNATURE
		responsiveshields-2.3-mc1.21.0.jar                |Responsive Shields            |responsiveshields             |2.3                 |DONE      |Manifest: NOSIGNATURE
		GD-BetterLoot_v1.0b3-1.21+_Fabric-Quilt-Forge-NeoF|GD BetterLoot                 |gd_betterloot                 |1.0b3               |DONE      |Manifest: NOSIGNATURE
	Crash Report UUID: 6e2c6683-6116-45af-a48a-b8d2476a82ff
	FML: 0.0
	Forge: net.minecraftforge:51.0.33

**Comment by daedalus4096** [comment]
Thank you for the additional information. I believe I've found the source of the crash and have implemented a fix, which will be included in the next release of Primal Magick.

**Comment by daedalus4096** [comment]
This bug is fixed in the upcoming v5.0.1-beta release. Thank you for the report! If you encounter this issue again, please file a new ticket.
